source/particles/particle_handler.cc

```diff
@@ -1376,7 +1376,8 @@ namespace Particles
                                                           reference_locations);
 
               if (GeometryInfo<dim>::is_inside_unit_cell(
-                    reference_locations[0]))
+                    reference_locations[0]),
+                  1e-12)
```

Comment:
You've got the parentheses wrong here. You write
```
  if (foo(), 1e-12)
```
where you mean
```
  if (foo(1e-12))
```
Nice catch by our warnings infrastructure :-)

Reply:
nice catch!  My bad, I should have seen it from the automatic indentation.
I'm not sure if 1e-12 is the best threshold here. It seems to work in most cases I tried.

Reply:
If you fix the parenthesis this looks good to me. I would suggest a threshold of 1e-10 as this is the default for `GridTools::find_active_cell_around_point`, that way #13202 combined with this PR should change as little as possible.